packages/site-demo/content/product/components/popover-dialog/index.mdx

```diff
@@ -0,0 +1,26 @@
+# PopoverDialog
+
+**PopoverDialogs are [Popovers](/product/components/popover) with a [dialog accessible pattern](https://www.w3.org/WAI/ARIA/apg/example-index/dialog-modal/dialog.html) applied to it.**
+
+<DemoBlock orientation="horizontal" demo="default" />
+
+### Accessibility concerns
+
+This component should be used when popovers contains complex content with multiple interactive elements.
+
+As this component uses the `dialog` accessible pattern, a few specific behaviors are applied:
+
+- When the popover is opened, focus will be set on either the element set using the `focusElement` prop or the first focusable element.
+Once the focus is set within the popover, it will be trapped and stay within the popover until it is closed.
+
+- A `label` must be defined using either `label`, `aria-label` or `aria-labelledby` props.
+Setting the `label` prop will apply an `aria-label` to the dialog but `aria-labelledby` can also be used to target
+a visible element as popover title.
```

Comment:
Some minor fix and also maybe we could recommend `aria-labelledby` more than `aria-label` ?
```suggestion
- A label must be defined using either `aria-label` or `aria-labelledby` props.
  Setting `aria-labelledby` is recommended as it target a visible label rather than adding an invisible `aria-label` on the dialog.
```

Reply:
Sure. I've also inverted the types priorities so that the first error to popup is for the missing `aria-labelledby` prop. :)